net/freeradius3/opnsense/service/conf/actions.d/actions_freeradius3.conf

```diff
@@ -0,0 +1,29 @@
+[start]
+command:/usr/local/opnsense/scripts/freeradius3/setup.sh;/usr/local/etc/rc.d/radiusd start
+parameters:
+type:script
+message:starting freeradius3
```

Comment:
In my opinion it should be written "FreeRADIUS" as this is how the project spells itself: http://freeradius.org/